### DISCUSSION TITLE
Rolling with advantage/disadvantage in statblocks?

### DISCUSSION THREAD

**Discussion by dBunkness** [Obsidian-TTRPG-Community/fantasy-statblocks#132] [Q&A] [answered]
I've been playing around with statblocks for a little bit now, and I can't see to find a way to allow rolling for advantage/disadvantage directly in the statblock.

E.G. -- the phrase `+2 to hit, 6 (1d6+8)`  allows for rolling a d20, and then your d6 for damage. If I change that to anything else like `2d20dh +2 to hit...` it will still only roll 1d20.

Is this possible?

Appreciate any help/feedback!

**Comment by valentine195** [comment, marked as answer]
The auto parsing won’t work here. 

You’ll have to create a layout (or a copy of the 5e one) and set the layout block to markdown, then enter a dice code block (like `dice: 2d20dh`)

Or create a new layout block with a custom dice callback (more complicated)

**Reply by dBunkness** [reply]
Thanks for the reply!

That will get the jo done for now! I may learn about the custom callbacks later on.

**Reply by sigrunixia** [reply]
> The auto parsing won’t work here.
> 
> You’ll have to create a layout (or a copy of the 5e one) and set the layout block to markdown, then enter a dice code block (like `dice: 2d20dh`)
> 
> Or create a new layout block with a custom dice callback (more complicated)

For Q&A/Documentation purposes, is this still the case?
